pages/index.tsx

```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
+              T-shirt dan jaket ReactJS Indonesia.
+            </Text>
+            <LinkButton ml="-16px" href="https://www.rumahkomunitas.com/react-indonesia" isExternal>
+              Dapatkan Segera
+            </LinkButton>
+          </Grid>
+        </Grid>
+      </Box>
+
+      <Box backgroundColor="lightBlue" paddingX="8.5em" paddingY="6em">
+        <Grid gap="2px" justifyContent="center" gridAutoFlow="row" textAlign="center">
+          <Heading as="h2" color="gray08" fontSize="20px" lineHeight="28px" fontWeight={300} textTransform="uppercase">
+            ingin belajar react?
+          </Heading>
+          <Heading as="h3" fontSize="48px" lineHeight="56px">
+            Materi Pembelajaran
+          </Heading>
+          <Text as="p" my="1.2em">
+            Beberapa konsep React memang terlihat janggal, tapi diluar itu React sangat mudah untuk dipelajari dan dipahami, baik mereka
+            yang sudah mahir dalam JavaScript modern ataupun yang baru saja memulai. Cobalah memulai dari salah satu materi di bawah.
+          </Text>
+          <Grid gridTemplateColumns="repeat(auto-fit, minmax(calc(296px), 1fr))" gap="24px" mt="24px">
+            {learningResources
+              .filter(resource => resource.featured)
+              .map((resource, i) => (
+                <PseudoBox
+                  key={i.toString()}
+                  role="group"
+                  backgroundColor="white"
+                  border="none"
+                  boxShadow="rgba(0, 0, 0, 0.25) 0px 2px 4px"
+                  minHeight="320px"
+                  borderRadius="8px"
+                  py="1em"
+                  px="1.5em"
+                  cursor="pointer"
+                  _hover={{ boxShadow: 'rgba(0, 0, 0, 0.25) 0px 6px 8px' }}
+                >
+                  <Link h="100%" href={resource.url} isExternal _hover={{ outline: 'none' }}>
+                    <Flex flexDirection="column" alignItems="flex-start" h="inherit">
+                      <Heading color="astronautBlue" as="h4" mb="4px" fontSize="1em" fontWeight={300} textTransform="uppercase">
+                        {resource.type}
+                      </Heading>
+                      <Heading textAlign="left" color="gray08" as="h5" fontSize="24px" fontWeight={600}>
+                        {resource.title}
+                      </Heading>
+                      <Text textAlign="left" as="p" mt="0.5em" mb="1.3em">
+                        {resource.description}
+                      </Text>
+                      <LinkButton
+                        _hover={undefined}
+                        _groupHover={{ backgroundColor: 'tangaroa' }}
+                        as="span"
+                        mt="auto"
+                        backgroundColor="astronautBlue"
+                        color="white"
+                      >
+                        Kunjungin Situs
```

Comment:
```suggestion
                        Kunjungi Situs
```

`Kunjungin` -> `Kunjungi`